src/App.tsx

```diff
@@ -1,18 +1,31 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
-import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const total = 42;
+  const [selectedPage, setSelectedPage] = useState(1);
+  const [selectedOption, setSelectedOption] = useState(3);
+
+  const items = Array.from({ length: 42 }, (_, index) => index + 1);
+  const options = [3, 5, 10, 15];
+
+  const lastItemOnPage = selectedPage * selectedOption;
+  const firstItemOnPage = lastItemOnPage - selectedOption;
+
+  const pageChangeHandler = (page: number, option: number) => {
+    setSelectedPage(page);
+    setSelectedOption(option);
```

Comment:
you should reset values to avoid this behavior
https://github.com/mate-academy/react_pagination/assets/117587463/545887ae-453d-4bf1-8877-d4a65b248ca0